src/generator/AutoRest.Ruby.Azure.Tests/RspecTests/paging_spec.rb

```diff
@@ -129,6 +129,21 @@
     expect(items.count).to eq(10)
   end
 
+  it 'should get multiple pages with fragmented next link' do

```

Comment:
should we be adding some more cases in swagger (paging.json) to account for paramater grouping with paging, and maybe some of the other cases we tried, to exercise our code?